backend/canisters/user_index/api/src/main.rs

```diff
@@ -38,6 +40,43 @@ fn main() {
     generate_candid_method!(user_index, unsuspend_user, update);
     generate_candid_method!(user_index, update_diamond_membership_subscription, update);
 
+    let directory = env::current_dir()
+        .unwrap()
+        .join("frontend/openchat-agent/tsBindings/userIndex");
+    if directory.exists() {
+        std::fs::remove_dir_all(&directory).unwrap();
+    }
+
+    generate_ts_method!(user_index, check_username);
```

Comment:
Anyway to automate this so it extracts all queries and updates without having to explicitly list them?

Reply:
Yeah we probably could. But there are some that we don't want to include which makes it harder to automate.